Kitodo-DataAccess/src/main/java/org/kitodo/dataaccess/format/xml/Space.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataaccess.format.xml;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Enum implementing the XML white space handling value constants.
+ *
+ * @see "http://www.w3.org/TR/xml/#sec-white-space"
+ */
+enum Space {
+
+    /**
+     * Signals that white-space processing is acceptable for this element.
+     */
+    DEFAULT("default") {
+        @Override
+        /**
+         * Trimming white space the XML way.
+         */
+        public String trim(String s) {
+            State state = State.INITIAL;
+            StringBuilder result = new StringBuilder(s.length());
+            for (int i = 0; i < s.length(); i++) {
+                int codePoint = s.codePointAt(i);
+                if (!WHITE_SPACE.contains(codePoint)) {
+                    if (state.equals(State.WHITE_SPACE)) {
+                        result.append(' ');
+                    }
+                    result.appendCodePoint(codePoint);
+                    state = State.TEXT;
+                } else if (state.equals(State.TEXT)) {
+                    state = State.WHITE_SPACE;
+                }
+            }
+            return result.toString();
+        }
+    },
+
+    /**
+     * Signals the intention that in that element the white space should be
+     * preserved.
+     */
+    PRESERVE("preserve") {
+        @Override
+        /**
+         * Trimming is doing nothing.
+         */
+        public String trim(String s) {
```

Comment:
1) Why is this method is needed but do nothing?
2) Short method parameter names should be avoided. Please use a longer and understandable name.

Reply:
The enum has got an interface, so each enum member implements that interface, and behaves differently. The one case does reduce white space (indents) as defined by XML specification, the other one does nothing (behaviour like in HTML `<pre>` tags)